src/Microsoft.Cci.Extensions/project.json

```diff
@@ -0,0 +1,36 @@
+﻿{
+    "dependencies": {
+        "System.AppContext": "4.0.0-beta-*",
```

Comment:
Would you please strip these down to just what is necessary? I want to understand the real dependencies this guy has.